Lib9c/Policy/BlockPolicySource.cs

```diff
@@ -234,11 +234,7 @@ public partial class BlockPolicySource
                     authorizedMinersPolicy,
                     defaultAlgorithm: chain => DifficultyAdjustment<NCAction>.BaseAlgorithm(
                         chain, BlockInterval, DifficultyStability, minimumDifficulty));
-            Func<Address, long, bool> isAllowedToMine = (address, index) => IsAllowedToMineRaw(
-                address,
-                index,
-                authorizedMinersPolicy,
-                permissionedMinersPolicy);
+            Func<Address, long, bool> isAllowedToMine = (address, index) => true;
```

Comment:
I guess we are unnecessary this lambda. right?

Reply:
Yes. This is an another temporary fix to run PBFT nodes ASAP///

Reply:
Why can't we clear the lambda?

Reply:
`BlockPolicy()` constructor takes `isAllowedToMine` parameter and it's default value is `null` not `true`.
https://github.com/planetarium/lib9c/blob/pbft/Lib9c/Policy/BlockPolicySource.cs#L260
https://github.com/planetarium/lib9c/blob/pbft/Lib9c/Policy/BlockPolicy.cs#L34

Reply:
LGTM